doc/source/batch-inference/doc_code/torch_image_batch_pretrained.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, List
+import numpy as np
+
+import torch
+from torchvision import transforms
+from torchvision.models import resnet18
+
+import ray
+from ray.train.torch import TorchCheckpoint, TorchPredictor
+from ray.train.batch_predictor import BatchPredictor
```

Comment:
Most of these imports seem unnecessary and are not used below.

Reply:
good catch, removed!